### ISSUE TITLE
Feature Request: "Clear Cache and Refresh" equivalent API Call

### ISSUE THREAD

**Issue by ghost** [looker-open-source/sdk-codegen#1230]
Hi,

I want to be able to "clear cache and refresh" a dashboard using the Python SDK, but couldn't see any option to do so. Is it possible to have this API call as a future feature?

Currently I'm using datagroups to clear cache via SQL trigger and then send emails after the cache's are updated daily. However, it would be much easier if I can use API calls to directly trigger "clear cache and refresh" of a specific dashboard.

**Comment by jrpadron** [comment]
Hello :) I have the same request, is there any news on this?

**Comment by NoamMosko** [comment]
Any updates on this??

**Comment by drstrangelooker** [comment]
Can you provide more details about what you want to do? For example, are you trying to render a dashboard as a pdf?

**Comment by NoamMosko** [comment]
Exactly! That is what we are doing, But for some reason some of the data we see is partial as if it didn't finish aggregating all numbers for all tiles.
I thought it might be due to cache issues. Thanks

**Comment by drstrangelooker** [comment]
This would be a product feature and would have to be added to the API endpoint. Once that happens it would be included in the SDK automatically when it is regenerated.

Can you please request this as a feature in the product?
